### ISSUE TITLE
GeoMakie example does not show geoaxis

### ISSUE THREAD

**Issue by rjdverbeek-tud** [JuliaPlots/GeoMakie.jl#55]
I am running the GeoMakie (v0.1.14) example (https://github.com/JuliaPlots/GeoMakie.jl/blob/master/README.md) in IJulia (Windows 10), but the geoaxis object is not shown. The rest, including the coastline is shown. What am I doing wrong?
![download](https://user-images.githubusercontent.com/54448042/91891541-7f659c00-ec91-11ea-9ba6-26753e487bf7.png)

```
using GeoMakie, Makie
AbstractPlotting.inline!(true)

lons = LinRange(-179.5, 179.5, 360)
lats = LinRange(-89.5, 89.5, 180)

field = [exp(cosd(l)) + 3(y/90) for l in lons, y in lats]

source = LonLat()
dest = WinkelTripel()

xs, ys = xygrid(lons, lats)
Proj4.transform!(source, dest, vec(xs), vec(ys))

scene = surface(xs, ys; color = field, shading = false, show_axis = false, scale_plot = false)

geoaxis!(scene, extrema(lons), extrema(lats); crs = (src = source, dest = dest,))

coastlines!(scene; crs = (src = source, dest = dest,))
```

**Comment by ali-ramadhan** [comment]
Also not seeing the `geoaxis` on Julia v1.5.2 + Makie v0.11.0 + GeoMakie v0.1.14.

**Comment by asinghvi17** [comment]
Should be fixed now, and will be properly fixed in #103
